common/sgx/tcbinfo.h

```diff
@@ -86,6 +136,14 @@ typedef struct _oe_parsed_tcb_info
 
 } oe_parsed_tcb_info_t;
 
+/*!
+ * Parse an oe_tcb_info_tcb_level_t struct to an oe_tcb_status_t type.
+ *
+ * @param[in] oe_tcb_level_status_t* The tcb_status to parse.
+ */
+oe_tcb_status_t oe_tcb_level_status_to_type(
+    const oe_tcb_level_status_t* tcb_status);
```

Comment:
```suggestion
    const oe_tcb_level_status_t* tcb_level_status);
```
If I understand the types (oe_tcb_level_status_t, oe_tcb_status_t) right, you pass _in_ a "tcb level status" and get _back_ a "tcb status".